cli/src/main/java/module-info.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Command-line interface (CLI) for validating both XML formats, converting from
+ * the event-based to the hierarchical format, and generating HTML reports
+ *
+ * @since 0.2.0
+ */
+module org.opentest4j.reporting.cli {
+
+    requires static org.apiguardian.api;
+    requires info.picocli;
+    requires java.desktop;
+    requires org.opentest4j.reporting.tooling.core;
+    requires org.slf4j;
+
+    exports org.opentest4j.reporting.cli;
```

Comment:
Would a targeted export directive suffice here?
```suggestion
    exports org.opentest4j.reporting.cli to info.picocli;
```

Reply:
I'm not sure :slightly_smiling_face:

Reply:
Well, not sure. I've marked them as `@API(status = EXPERIMENTAL)`.

Reply:
Good enough, I guess. And with `EXPERIMENTAL` we may restrict the range in a later release.